src/components/progressBar/ProgressBar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {SafeAreaView, StatusBar, Text, View} from 'react-native';
+import {styles} from './progressBar.style';
+
+export const ProgressBar = ({value = '5%', max = '100'}) => {
+  return (
+    <View style={styles.container}>
+      <View style={[styles.frontBar, {width: `${max}%`}]} />
+      <View style={[styles.backBar, {width: `${value}%`}]} />
```

Comment:
porgressIndicaterContainer - would is a better way.

Reply:
added the changes for v.2